src/ui/components/pages/MyServices/MyServicesSavedConfigs/MyServicesSavedConfig/MyServicesSavedConfig.tsx

```diff
@@ -82,33 +82,44 @@ export const MyServicesSavedConfig = memo((props: Props) => {
                     {friendlyName}
                 </Text>
             </div>
-            <div className={classes.linkAndEditButtonWrapper}>
-                <IconButton
-                    className={classes.linkIcon}
-                    iconId="link"
-                    onClick={onLinkClick}
-                />
+            {editLink.href.includes("version") && (
```

Comment:
Ok haha,  
Sorry but this is to much of a hack. 😄  
There no ovious good solution here but let's try to keep it under control.